Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,588 @@
+        type: 'stringWithSelector',
+        paramLabel: _('Alignment'),
+        options: `["left", "right", "center"]`,
+      },
+      {
+        functionName: 'WordWrap',
+        description: _('word wrap'),
+        iconPath: 'res/actions/scaleWidth24',
+        type: 'boolean',
+        paramLabel: _('Word wrap'),
+      },
+      {
+        functionName: 'WrappingWidth',
+        description: _('wrapping width'),
+        iconPath: 'res/actions/scaleWidth24',
+        type: 'number',
+        paramLabel: _('Wrapping width'),
+      },
+    ];
+
+    addSettersAndGettersToObjectHelper(
+      object,
+      setterAndGetterProperties,
+      'BBText'
+    );
+
+    return extension;
+  },
+
+  /**
+   * You can optionally add sanity tests that will check the basic working
+   * of your extension behaviors/objects by instanciating behaviors/objects
+   * and setting the property to a given value.
+   *
+   * If you don't have any tests, you can simply return an empty array like this:
+   * `runExtensionSanityTests: function(gd, extension) { return []; }`
+   *
+   * But it is recommended to create tests for the behaviors/objects properties you created
+   * to avoid mistakes.
+   */
+  runExtensionSanityTests: function(gd, extension) {
+    return [];
+  },
+  /**
+   * Register editors for objects.
+   *
+   * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change.
+   */
+  registerEditorConfigurations: function(objectsEditorService) {
+    objectsEditorService.registerEditorConfiguration(
+      'BBText::BBText',
+      objectsEditorService.getDefaultObjectJsImplementationPropertiesEditor({
+        helpPagePath: '/objects/bbtext_object',
+      })
+    );
+  },
+  /**
+   * Register renderers for instance of objects on the scene editor.
+   *
+   * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change.
+   */
+  registerInstanceRenderers: function(objectsRenderingService) {
+    const RenderedInstance = objectsRenderingService.RenderedInstance;
+    const PIXI = objectsRenderingService.PIXI;
+
+    /**
+     * Renderer for instances of BBText inside the IDE.
+     *
+     * @extends RenderedBBTextInstance
+     * @class RenderedBBTextInstance
+     * @constructor
+     */
+    function RenderedBBTextInstance(
+      project,
+      layout,
+      instance,
+      associatedObject,
+      pixiContainer,
+      pixiResourcesLoader
+    ) {
+      RenderedInstance.call(
+        this,
+        project,
+        layout,
+        instance,
+        associatedObject,
+        pixiContainer,
+        pixiResourcesLoader
+      );
+
+      const BBTextStyles = {
+        default: {
+          fontFamily: 'Arial',
+          fontSize: '24px',
+          fill: '#cccccc',
+          tagStyle: 'bbcode',
+          wordWrap: true,
+          wordWrapWidth: 250,
+          align: 'left',
+        },
+      };
+
+      const MultiStyleText = objectsRenderingService.requireModule(
```

Comment:
Move this after `const PIXI = objectsRenderingService.PIXI`.
Consider `requireModule` as a "require": you do this only once to fetch the module at the top of your file, or at least, in this case, at the top of the `registerInstanceRenderers` function so that you don't pay for searching for the module in the constructor of every single instance that will be shown on the scene.